content/module-reference/processing-modules/color-calibration.md

```diff
@@ -0,0 +1,100 @@
+
+ - _same as pipeline (D50)_, where the D65 illuminant white balance parameters for the camera that was set by the _white balance_ module is adjusted to a D50 illuminant
+ -  one of the CIE standard illuminants (daylight, incandescent, fluorescent, equi-energy, or black body), or a non-standard _LED light_ illuminant.
+ - _custom_, in which case the illuminant color can be set using the color picker icon, or by setting the hue and chroma (saturation) in LCh perceptual color space using the sliders
+ - _(AI) detect from image surfaces_, where the white balance is auto-detected, placing a greater weighting on colors in the image close to grey
+ - _(AI) detect from image edges_, where the auto-balance is auto-detected, assuming the average differences across edges in the image are achromatic.
+ - _as shot in camera_, in which case the illuminant will be calculated based on the white balance settings provided by the camera.
+
+temperature
+: For the standard illuminants, the _temperature_ slider allows you to adjust the color temperature of the illuminant. Moving the slider to the right means the illuminant is assumed to have been more blue, which will make the white-balanced picture appear warmer/more red after the white balance compensation in applied. Moving the slider to the left sets the illuminant to a more red color, which makes the image appear cooler/mor blue after compensation.
+
+hue
+: For custom white balance, the _hue_ slider allows the hue of illuminant color in LCh color space to be manually set.
+
+chroma
+: For custom white balance, the _chroma_ slider allows the Chroma (or saturation) of illuminant color in LCh color space to be manually set.
+
+gamut compression
+: Allows the color gamut to be compressed in order to bring out-of-gamut colors back into gamut, and avoid nasty "color clipping".
+
+## R, G and B tabs
+
+These tabs allow the red, green and blue channel outputs for each pixel to be make up of a weighted sum of the red, green and blue channel inputs. Each tab has 3 sliders, and there are 3 tabs, so the sliders are effectively setting the coefficients of a 3x3 matrix, just like in the _channel mixer_ module
+
+input red/green/blue
+: These sliders set the mix of how heavily the input R, G and B channels influence the output channel relating to the tab concerned.
+
+normalize channels
+: If this checkbox is selected, then the coefficients set on the sliders for this tab are normalised to try to preserve the overall brightness of this channel in the final image as compared to the input image. 
+
+## colorfulness tab
+
+input red/green/blue
+: these sliders allow the saturation of certain colors in the image to be adjusted, depending on how much red, green or blue components those colors contain. For example, adjusting the _input red_ slider will affect the saturation of colors containing a lot of red much more than colors containing only a small amount of red. 
```

Comment:
is it the saturation of "colors" in the image or the saturation of "pixels"?

Reply:
To be clear, we are talking about colour saturation of pixels, where a pixel is an RGB triplet. Since an image is made up of pixels, this translates into colour saturation within the image.

Reply:
I'm just saying maybe this might read better as "allow the saturation of pixels to be adjusted, depending on how much..."

Reply:
implemented comment in latest commit